index.js

```diff
@@ -745,9 +745,57 @@ const runTests = async () => {
   process.chdir(workingDirectory)
   return cypress
     .run(cypressOptions)
+    .then(generateSummary)
     .then(onTestsFinished, onTestsError)
 }
 
+const generateSummary = async (testResults) => {
+  console.log(
```

Comment:
I know this is just a draft, but I am assuming you just added this for testing purposes? or is this console.log necessary?